src/app/view/view-controls/view-controls.component.html

```diff
@@ -1,28 +1,24 @@
-<form class="form-inline d-flex">
-  <span i18n="@@view.perspective" class="col-form-label font-weight-bold">Perspective</span>:
-  <div class="form-control dropdown mx-3 p-0">
-    <div class="dropdown-toggle py-1 px-2" id="dropdownMenuButton" data-toggle="dropdown"
-         aria-haspopup="true" aria-expanded="false">
+<form class="form-inline d-flex flex-nowrap">
+  <div class="form-control dropdown mr-2 p-0" title="Perspective" i18n-title="@@view.perspective">
+    <div class="dropdown-toggle py-1 px-2 d-flex flex-nowrap align-items-center"
+         id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
       <i class="fa-fw {{getIconForPerspective(view.perspective)}}"></i>
-      <span class="ml-1">{view.perspective, select, postit {Post-it} search {Search} table {Table}}</span>
+      <span class="ml-1" i18n="@@view.perspective">{view.perspective, select, postit {Post-it} search {Search} table {Table}}</span>
     </div>
     <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
       <div class="dropdown-item" *ngFor="let perspective of perspectives()"
            (click)="onSelectPerspective(perspective)">
         <i class="fa-fw {{getIconForPerspective(perspective)}}"></i>
-        <span class="ml-1">{perspective, select, postit {Post-it} search {Search} table {Table}}</span>
+        <span class="ml-1" i18n="@@view.perspective">{perspective, select, postit {Post-it} search {Search} table {Table}}</span>
       </div>
     </div>
   </div>
-  <label for="viewName"
-         class="col-form-label font-weight-bold">
-    <span i18n="@@view.name">Name</span>:
-  </label>
   <input #viewName
          id="viewName"
          name="viewName"
+         title="Name" i18n-title="@@view.name"
```

Comment:
Isn't _Name_ too abstract? I think user might be confused what name it is. What about calling it _View Name_?

Reply:
You are right. I just copied the old version.